File: src/main/java/br/com/rfasioli/micrometermetrics/configuration/MetricsConfiguration.java
package br.com.rfasioli.micrometermetrics.configuration;

import io.micrometer.core.instrument.MeterRegistry;
import org.springframework.boot.actuate.autoconfigure.metrics.MeterRegistryCustomizer;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class MetricsConfiguration {
  @Bean
  MeterRegistryCustomizer<MeterRegistry> metricsCommonTags() {
    return registry -> registry.config().commonTags("application", "MicrometerMetricsDemo");
  }
}

File: src/main/kotlin/br/com/rfasioli/micrometermetrics/configuration/MetricsConfiguration.kt
package br.com.rfasioli.micrometermetrics.configuration

import io.micrometer.core.instrument.MeterRegistry
import org.springframework.boot.actuate.autoconfigure.metrics.MeterRegistryCustomizer
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class MetricsConfiguration {
    @Bean
    fun metricsCommonTags(): MeterRegistryCustomizer<MeterRegistry> =
        MeterRegistryCustomizer { registry: MeterRegistry ->
            registry.config().commonTags("application", "MicrometerMetricsDemo")
        }
}
